docs/crictl.md

```diff
@@ -0,0 +1,160 @@
+        "attempt": 1,
+        "uid": "hdishd83djaidwnduwk28bcsb"
+    },
+    "linux": {
+    }
+}
+
+# crictl runs sandbox-config.json
+e1c83b0b8d481d4af8ba98d5f7812577fc175a37b10dc824335951f52addbb4e
+# crictl sandboxes
+SANDBOX ID                                                         NAME                STATE
+e1c83b0b8d481d4af8ba98d5f7812577fc175a37b10dc824335951f52addbb4e   nginx-sandbox       SANDBOX_READY
+# crictl inspects e1c83
+  ... displays information about the pod and the sandbox pause container.
+```
+* Note: As shown above, you may use truncated IDs if they are unique.
+* Other commands to manage the pod include `stops ID` to stop a running pod and
+`rms ID` to remove a pod sandbox.
+
+## Create and Run a Container in a Sandbox (using a config file)
+```shell
+# cat sandbox-config.json
+{
+    "metadata": {
+        "name": "nginx-sandbox",
+        "namespace": "default",
+        "attempt": 1,
+        "uid": "hdishd83djaidwnduwk28bcsb"
+    },
+    "linux": {
+    }
+}
+
+# cat container-config.json
+{
+  "metadata": {
+      "name": "busybox"
+  },
+  "image":{
+      "image": "busybox"
+  },
+  "command": [
+      "top"
+  ],
+  "linux": {
+  }
+}
+
+# crictl create e1c83 container-config.json sandbox-config.json
+0a2c761303163f2acaaeaee07d2ba143ee4cea7e3bde3d32190e2a36525c8a05
+# crictl ps
+CONTAINER ID                                                       CREATED             STATE               NAME
+0a2c761303163f2acaaeaee07d2ba143ee4cea7e3bde3d32190e2a36525c8a05   1 minutes ago       CONTAINER_CREATED   busybox
+# crictl start 0a2c
+0a2c761303163f2acaaeaee07d2ba143ee4cea7e3bde3d32190e2a36525c8a05
+# crictl ps
+CONTAINER ID                                                       CREATED             STATE               NAME
+0a2c761303163f2acaaeaee07d2ba143ee4cea7e3bde3d32190e2a36525c8a05   2 minutes ago       CONTAINER_RUNNING   busybox
+# crictl inspect 0a2c7
+  ... show detailed information about the container
+```
+## Exec a Command in the Container
+```shell
+# crictl exec -i -t 0a2c ls
+bin   dev   etc   home  proc  root  sys   tmp   usr   var
+```
+## Display Stats for the Container
+```shell
+# crictl stats
+CONTAINER           CPU %               MEM                 DISK                INODES
+0a2c761303163f      0.00                991.2kB             16.38kB             6
+```
+* Other commands to manage the container include `stop ID` to stop a running
+container and `rm ID` to remove a container.
+## Display Version Information
+```shell
+crictl version
+Version:  0.1.0
+RuntimeName:  cri-containerd
+RuntimeVersion:  1.0.0-alpha.1-167-g737efe7-dirty
+RuntimeApiVersion:  0.0.0
+```
+## Display Status & Configuration Information about Containerd & CRI-Containerd
+```shell
+crictl info
+{
+  "status": {
+    "conditions": [
+      {
+        "type": "RuntimeReady",
+        "status": true,
+        "reason": "",
+        "message": ""
+      },
+      {
+        "type": "NetworkReady",
```

Comment:
Just for my knowledge , Is this true if CNI plugin is installed and running ?

Reply:
yes

Reply:
Yeah, if cni plugin and configuration are installed.